programs/client/Client.cpp

```diff
@@ -106,7 +106,9 @@ void Client::processError(const String & query) const
 std::vector<String> Client::loadWarningMessages()
 {
     std::vector<String> messages;
-    connection->sendQuery(connection_parameters.timeouts, "SELECT message FROM system.warnings", "" /* query_id */,
+    connection->sendQuery(connection_parameters.timeouts,
```

Comment:
@vitlibar Maybe client should check server version/revision to see if function is supported, to make new client compatible with older servers?

Reply:
I'll do that

Reply:
https://github.com/ClickHouse/ClickHouse/pull/39413